piyushgarg-dev/piyushgargdev-nextjs
# feat: Implemented hover On Social Media buttons In Contact page.

What does this PR do?
This PR adds the hover effect on the social media icons in Contact page.

Fixes #131 

https://github.com/piyushgarg-dev/piyushgargdev-nextjs/assets/110960043/8293d03f-0461-4742-a9b3-7dbcae5c641e


## Type of change
- New feature (non-breaking change which adds functionality)


## How should this be tested?

How should this be tested?
1. Open project locally at port 3000
2. Locate the social media icons In the contact Page.
3. Hover on the icons. The color will get changed to theme color.

## Mandatory Tasks
- [X ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

Thank You :)
